docs/build-the-docs.md

```diff
@@ -24,7 +24,7 @@ docker run -ti --rm \
       --cache-dir /antora/cache/ \
       --generator=./generate-site.js \
       --stacktrace \
-      --url <URL or path to your Antora instance> \
+      --url https://owncloud-docs.netlify.com/ \
```

Comment:
`--url <URL of your webserver for local testing>` should be better, the production URL is part of the `site.yml`

Reply:
Okay. I guess I misunderstood you previously.

Reply:
Fixed.